BD-2088: Document meta throughput limit in WhatsApp FAQ (#5365)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/whatsapp/faqs.md

#### Can my messaging limit decrease?
Yes, if your phone number quality rating drops too low, you are at risk of WhatsApp decreasing your messaging limit. Braze recommends you subscribe and be notified of quality-related updates from WhatsApp, including updates to your phone number status and messaging limit level. You can subscribe to notifications directly in the WhatsApp Manager dashboard. 

#### What is the Meta throughput limit?

> It is important to note that Meta has their own throughput limit seperate from the WABA messaging limit. The default limit the cloud API supports is 80 messages per second. If you think your campaigns will exceed this limit, you can [request](https://developers.facebook.com/docs/whatsapp/cloud-api/overview/#throughput) for your limit be increased. Note that it is recommended to submit this request at least three days in advance of campaign sends.

Reviewer: Some typos and a few little suggestions for flow 👀 
```suggestion
Meta has their own throughput limit separate from the WABA messaging limit. The default limit the cloud API supports is 80 messages per second. If you think your campaigns will exceed this limit, you can [request](https://developers.facebook.com/docs/whatsapp/cloud-api/overview/#throughput) for your limit to be increased. We recommend that you submit this request at least three days in advance of campaign sends.
```
